CMU-313/fall23-nodebb-formula1
# Insert bug data form into database

Resolves #6 

- Creates jQuery file that creates object with form data and posts it to api
- Write controller contains new function submitData 
- Uses the bug controller's create function to insert into the database
- Translates src/bugs/create.js to ts
- Creates bid (bug id) as unique identifier for bug reports
- Inserts email as a field

Test:

- pull code
- run ./nodebb build && ./nodebb start
- go to bug report form page by clicking on the bug icon in nav
- Fill out form (name, email, date, and description are required)
- check that the values of the form are printed in ./nodebb log
